src/libcore/fmt/mod.rs

```diff
@@ -269,6 +269,22 @@ impl<'a> Display for Arguments<'a> {
 
 /// Format trait for the `:?` format. Useful for debugging, all types
 /// should implement this.
+///
+/// Generally speaking, you should just `derive` a `Debug` implementation.
```

Comment:
Probably worth mentioning the debug builder methods for use by manual implementations (`Formatter::debug_<foo>`)